qdrant_client/local/distances.py

```diff
@@ -49,12 +53,12 @@ def __init__(self, context_pairs: List[ContextPair]):
         self.context_pairs = context_pairs
 
 
-QueryVector = Union[DiscoveryQuery, ContextQuery, RecoQuery, types.NumpyArray, SparseVector]
-
-
-class DistanceOrder(str, Enum):
-    BIGGER_IS_BETTER = "bigger_is_better"
-    SMALLER_IS_BETTER = "smaller_is_better"
+QueryVector = Union[
```

Comment:
Only a suggestion, I'll leave it to your criteria
```suggestion
DenseQueryVector = Union[
```